GVRf/Framework/backend_oculus/src/main/java/org/gearvrf/OvrMonoscopicViewManager.java

```diff
@@ -126,32 +140,131 @@
             mViewportY = (screenHeightPixels / 2) - (fboHeight / 2);
         }
 
+        sampleCount = gvrActivity.getAppSettings().getEyeBufferParams().getMultiSamples();
+
+        try {
+            Process proc = Runtime.getRuntime().exec(new String[]{"/system/bin/getprop", "debug.gearvrf.vulkan"});
```

Comment:
I see
1. Vulkan-specific code in backend-oculus. I'd like to understand the design in detail, explaining why it is necessary and can't be hidden somewhere else.
2. Reading a property directly, instead of using our numerous property-reading routines (that need to be consolidated, and using yet another way to read properties is wrong)
3. Something that requires at least storage-read permission and may not work at all on user build phone